components/patterns/form/_form.scss

```diff
@@ -170,3 +171,33 @@ $form-widths: (
 .ajax-enabler {
   display: none;
 }
+
+.form__element__error {
+  bottom: -1.7em;
+  color: $tscRed;
+  display: none;
+  font-size: fs(xxxs);
+  left: 0;
+  margin: 0;
+  position: absolute;
+  white-space: nowrap;
+
+  @include respond-to(breakpoint(s)) {
+    bottom: -2em;
+    font-size: fs(xxs);
+  }
+}
```

Comment:
As being positioned absolute, it looks like a two-line error message would overlap with the text underneath the form. Is that true?

Reply:
yep, therefore `whitespace: nowrap` – but we also know what the error messages are so we don't have to worry too much

Reply:
How does it behave on mobile? Guess you will be having all inputs underneath each other

Reply:
yeah, it looks fine because the font size is decreased – but I'll check again to make sure there aren't issues